workers/unity/Packages/com.improbable.gdk.core/Config/ConnectionConfig.cs

```diff
@@ -77,13 +77,26 @@ public ConnectionParameters CreateConnectionParameters()
         /// <exception cref="ConnectionFailedException">
         ///     Thrown if configValue is null or empty.
         /// </exception>
-        protected void ValidateConfig(string configValue, string configName)
+        protected void ValidateString(string configValue, string configName)
```

Comment:
ValidateStringIsSet / ValidateStringNotEmpty?

Reply:
I've removed this method altogether.